VideoLocker/res/values/strings.xml

```diff
@@ -28,8 +28,14 @@
     <string name="share_application_picture_url">https://www.edx.org/sites/default/files/theme/edx-logo-header.png</string>
     <string name="share_application_description">Free online courses</string>
 
-    <!-- Settings Page -->
+    <!-- Course Sharing -->
+
+    <!-- Share menu title -->
+    <string name="share_course_popup_header">Share this course</string>
```

Comment:
I think you should use "Share this course" as the contentDescription, but then use "Share this course via…" as the popup menu header. (Partly as a hack to increase the width of the popup as it's determined by the header label :( ...)